src/styles/_catalog.scss

```
@@ -20,3 +17,27 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width:1024px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    grid-template-rows: auto auto auto auto;
+  }
+}
+
+@media (min-width:768px) and (max-width:1023px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto auto;
+  }
+}
+
+@media (min-width:488px) and (max-width:767px) {
+  .catalog {
+    display: grid;
```

no need to set display grid on each breakpoint.
just set one time above for `.catalog`
also redundant to use `and` enough min-width

```suggestion
@media (min-width:488px) {
  .catalog {
```